powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/ActivationFlagsServiceBehavior.java

```diff
@@ -191,6 +193,7 @@ public RemoveActivationFlagsResponse removeActivationFlags(String activationId,
                 .build();
         audit.log(AuditLevel.INFO, "Removing activation flags: {} from activation {}", auditDetail, activationFlags, activationId);
         activation.getFlags().removeAll(activationFlags);
+        activation.setTimestampLastChange(new Date());
```

Comment:
- [ ] As discussed with @petrdvorak, this change is not enough. It has to be connected with ActivationHistory.